examples/chart/teleport-cluster/values.yaml

```
@@ -63,7 +63,24 @@ authentication:
   # Second factor requirements for users of the Teleport cluster.
   # Controls the `auth_config.authentication.second_factor` field in `teleport.yaml`.
   # Possible values are 'off', 'on', 'otp', 'optional' and 'webauthn'.
-  secondFactor: "otp"
+  #
+  # WARNING:
+  #   If you set `publicAddr` for users to access the cluster under a domain different
+  #   than [`clusterName`](#clusterName) you must manually set the webauthn [Relying
+  #   Party Identifier (RP
+  #   ID)](https://www.w3.org/TR/webauthn-2/#relying-party-identifier). If you don't,
+  #   RP ID will default to `clusterName` and users will fail to register second
+  #   factors.
```

```suggestion
  #   to clusterName you must manually set the webauthn Relying
  #   Party Identifier (RP ID) - https://www.w3.org/TR/webauthn-2/#relying-party-identifier
  #   If you don't, RP ID will default to `clusterName` and users will fail
  #   to register second factors.
```

I don't think anchors/links will work in this file and the formatting looks weird